### DISCUSSION TITLE
Possible to use an external c-lightning node

### DISCUSSION THREAD

**Discussion by so7ow** [btcpayserver/btcpayserver#5611] [Technical Support] [answered]
I see in https://github.com/btcpayserver/btcpayserver/issues/498 from a few years ago, using an external c-lightning node is not recommended. Is that still the case, or has there been any progress on this front?

**Comment by dennisreimann** [comment, marked as answer]
We are planning to add the new CLN REST API and allow for integration that way. No ETA, but I need that for myself as well.
